bundle/edu.gemini.catalog/src/main/java/jsky/catalog/RangeSearchCondition.java

```diff
@@ -113,12 +85,12 @@ public boolean isTrueFor(Comparable val) {
                     && (_maxVal == null || _maxVal.compareTo(val) >= 0));
         }
 
-        if (_minInclusive && !_maxInclusive) {
+        if (_minInclusive) {
             return ((_minVal == null || _minVal.compareTo(val) <= 0)
                     && (_maxVal == null || _maxVal.compareTo(val) > 0));
         }
 
-        if (!_minInclusive && _maxInclusive) {
+        if (_maxInclusive) {
             return ((_minVal == null || _minVal.compareTo(val) < 0)
                     && (_maxVal == null || _maxVal.compareTo(val) >= 0));
         }
```

Comment:
That case is addressed on L83 just above.
(Yeah, I admit this was confusing as it didn't appear directly in the diff.)
The only reason I caught it was because IDEA was squawking about boolean conditions always being true.